src/seedu/addressbook/commands/Command.java

```diff
@@ -36,6 +36,10 @@ public static String getMessageForPersonListShownSummary(List<? extends ReadOnly
         return String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, personsDisplayed.size());
     }
 
+    public static String getMessageForPersonSortedShownSummary(List<? extends ReadOnlyPerson> personsDisplayed) {
```

Comment:
Do you think that changing the method name to `getSummaryForSortedPersons` is clearer?

Reply:
Yes